clients/hadoopfs/pom.xml

```diff
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>io.lakefs</groupId>
             <artifactId>api-client</artifactId>
-            <version>0.1.0</version>
+            <version>0.40.3-RC.0</version>
```

Comment:
This should be 0.1.0 right?

Reply:
Nope, but I bumped it to be the release version.  (We may revisit this decision later).